packages/crypto/src/secp256k1.rs

```diff
@@ -206,7 +206,7 @@ mod tests {
         .unwrap());
 
         // Wrong message fails
-        let bad_message_hash = Sha256::new().chain([MSG, "\0"].concat()).finalize();
+        let bad_message_hash = Sha256::digest([MSG, "\0"].concat().as_bytes());
```

Comment:
Isn't `Sha256::new().chain(MSG).chain("\0").finalize();` the better way to write this, avoiding a copy of the slices?

Reply:
As the name implies, use it! :-) Will change.